packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -57,64 +58,64 @@ import { Solarray } from "scripts/libraries/Solarray.sol";
 // Additionally, we intentionally use "Input" and "Output" terminology to clearly distinguish these
```

Comment:
the documentation above makes several references to 'structs' and should be updated where necessary.

Reply:
Removed in 1cbf77e1a